jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -681,26 +677,20 @@ private static void arraycopy(Object[] A1, int offset1, Object[] A2, int offset2
  */
 public static native long currentTimeMillis();
 
-/*[IF OpenJ9-RawBuild]*/
-	/* This is a JCL native required only by OpenJ9 raw build.
-	 * OpenJ9 raw build is a combination of OpenJ9 and OpenJDK binaries without JCL patches within extension repo.
-	 * Currently OpenJ9 depends on a JCL patch to initialize platform encoding which is not available to raw build.
-	 * A workaround for raw build is to invoke this JCL native which initializes platform encoding.
-	 * This workaround can be removed if that JCL patch is not required.
-	 */
+/*[IF JAVA_SPEC_VERSION == 11]*/
 private static native Properties initProperties(Properties props);
-/*[ENDIF] OpenJ9-RawBuild */
+/*[ENDIF] JAVA_SPEC_VERSION == 11 */
 
 /**
  * If systemProperties is unset, then create a new one based on the values
  * provided by the virtual machine.
  */
 @SuppressWarnings("nls")
 private static void ensureProperties(boolean isInitialization) {
-/*[IF OpenJ9-RawBuild]*/
-	// invoke JCL native to initialize platform encoding
-	initProperties(new Properties());
-/*[ENDIF] OpenJ9-RawBuild */
+	/*[IF JAVA_SPEC_VERSION == 11]*/
+	Properties jclProps = new Properties();
+	initProperties(jclProps);
+	/*[ENDIF] JAVA_SPEC_VERSION == 11 */
 
 /*[IF JAVA_SPEC_VERSION > 11]*/
```

Comment:
While you're here, can you please indent lines 695, 697 and 699 (like most of the rest of this method)?